packages/mobile/src/verify/VerificationInterstitialScreen.tsx

```diff
@@ -19,7 +20,7 @@ const AnimatedCircle = () => (
 )
 
 class VerificationInterstitialScreen extends React.Component<WithTranslation> {
-  static navigationOptions = { gestureEnabled: false, header: null }
+  static navigationOptions = noHeaderGestureDisabled
```

Comment:
Does defining navigation options like this still work as expected?

Reply:
Yes, tested successfully.